src/components/Option.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import TypeAhead from "../components/TypeAhead.svelte";
+  import result from "../components/TypeAhead.svelte";
+  import results from "../components/TypeAhead.svelte";
+  import value from "../components/TypeAhead.svelte";
+  import { createEventDispatcher } from "svelte";
+  const dispatch = createEventDispatcher();
+  export let id = results.id;
```

Comment:
I think you could just leave this as `export let id;`, since the id comes in from the Option declaration in Typeahead.svelte.